main.go

```diff
@@ -1,7 +1,41 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type Task struct {
+	ID          int    `json:"id"`
+	Title       string `json:"title"`
+	Description string `json:"description"`
+	DueDate     string `json:"dueDate"`
+}
+
+var taskList []Task
+
+func PostTask(c *gin.Context) {
+	var task Task
+
+	if err := c.ShouldBindJSON(&task); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+
+	task.ID = len(taskList) + 1
+	taskList = append(taskList, task)
+	fmt.Println(taskList)
+
+	c.JSON(http.StatusCreated, task)
+
+}
 
 func main() {
-	fmt.Println("Hola mundo")
+
+	r := gin.Default()
+	r.POST("/task", PostTask)
```

Comment:
```suggestion
	r.POST("/task", CreateTaskHandler)
```

Reply:
the same that previously request

Reply:
I also change the other function`s name 
GetTask to GetTaskHandler. Line 36, 54, 55